src/hooks/useTxNotifications.ts

```diff
@@ -6,15 +6,17 @@ import { AppRoutes } from '@/config/routes'
 import useSafeInfo from './useSafeInfo'
 import { useCurrentChain } from './useChains'
 
-const TxNotifications: Partial<Record<TxEvent, string>> = {
+const TxNotifications = {
   [TxEvent.SIGN_FAILED]: 'Signature failed. Please try again.',
   [TxEvent.PROPOSED]: 'Your transaction was successfully proposed.',
   [TxEvent.PROPOSE_FAILED]: 'Failed proposing the transaction. Please try again.',
   [TxEvent.SIGNATURE_PROPOSED]: 'You successfully signed the transaction.',
   [TxEvent.SIGNATURE_PROPOSE_FAILED]: 'Failed to send the signature. Please try again.',
   [TxEvent.EXECUTING]: 'Please confirm the execution in your wallet.',
-  [TxEvent.MINING]: 'Your transaction is being mined.',
-  [TxEvent.MINED]: 'Your transaction was successfully mined and is now being indexed.',
+  [TxEvent.PROCESSING]: 'Your transaction is being mined.',
+  [TxEvent.PROCESSING_MODULE]:
+    'Your transaction has been submitted and will appear in the interface only after it has been successfully mined and indexed.',
+  [TxEvent.PROCESSED]: 'Your transaction was successfully mined and is now being indexed.',
```

Comment:
Maybe we should adjust these to say "mined/validated"?

Reply:
When I asked to change the wording, I meant the notification message itself, not the enum props.

Reply:
Lets leave the enum naming how it was and only update the messages.

Reply:
done